### PR TITLE
Add line about robot story

### PR DESCRIPTION
**Pull request by Shamwow16**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2564)

### CODE REVIEW

File: english/marshmallow.md [hunk 1]
```diff
@@ -538,3 +538,5 @@ And you plug-in to [play some music](play-music/playmusic.md).
 The next thing you know, you're up all night in Croatia mixing a new playlist
 
 You remember the story of a [robot](happy-robot/happy-robot.md).
+
+You go to the robot shelter and get a robot that pours your favorite [wine] (wine/wine.md) in a decanter.
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**